app/decorators/repository_decorator.rb

```diff
@@ -8,12 +8,30 @@ def self.collection_decorator_class
   end
 
   def languages
-    return '' unless language
+    return [] unless language
 
-    language.split(',').first(3).to_sentence
+    language.split(', ').first(3).map(&:strip)
+  end
+
+  def issues_formatted
+    format_number(model.issues)
+  end
+
+  def stars_formatted
+    format_number(model.stars)
   end
 
   def name
     link[/[\w_-]+$/]
   end
+
+  private
+
+  def format_number(number)
+    if number >= 1000
+      "#{(number/1000).round(1)}K"
+    else
+      number
+    end
+  end
```

Comment:
On rails we have a method called `number_to_human`:

https://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_human

It helps to format this kind of approach.

Maybe something like:

```suggestion
  def format_number(number)
    number_to_human(number, precision: 1, units: {thousand: "k", billion: "kk"})
  end
```

It's also apply the i18n thing

Reply:
Awesome! I did not know that, thanks for the tip!